lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -52,8 +52,10 @@ function backtickCodeBlock(data) {
         if (options.exclude_languages) {
           if (options.exclude_languages.includes(options.lang)) {
             // Only wrap with <pre><code class="lang"></code></pre>
+            // If you can't get the proper output.
+            // Please refer to: https://github.com/hexojs/hexo-util/blob/master/lib/highlight.js
+            // or update hexo-util to the latest version.
```

Comment:
I forgot to mention that hexo-util is tightly integrated into hexo (#3888, #3829, #3728), as in they are always upgraded together.

Since this feature will only be implemented (I don't mind, but I don't know if maintainer will approve) in future version and not backported (thus, https://github.com/hexojs/hexo-util/pull/138 will be included), you don't have to worry about compatibility with older hexo-util. In fact, some hexo components may even error out if latest hexo-util is not used.

Reply:
Well, I delete those comments and rebase with this repo's master. Do I need to compress the commits?

Reply:
that's fine, I can do that when merging.